test/authentication/authentication.test.js

```diff
@@ -267,135 +271,175 @@ describe('auth0.authentication', function() {
       });
+      it('returns ssoData object with lastUsedConnection and idtokenpayload.name when there is no idtokenpayload.email', function(
+        done
+      ) {
+        this.auth0.getSSOData(function(err, result) {
+          expect(err).to.be(null);
+          expect(result).to.be.eql({
+            lastUsedConnection: { name: 'lastUsedConnection' },
+            lastUsedUserID: 'the-user-id',
+            lastUsedUsername: 'last-used-user-name',
+            lastUsedClientID: '...',
+            sessionClients: ['...'],
+            sso: true
+          });
+          done();
+        });
 
-      this.webAuthSpy.checkSession.lastCall.args[1]({
-        error: 'consent_required',
-        error_description: 'foobar'
-      });
-    });
-    it('returns ssoData object with lastUsedConnection and idtokenpayload.name when there is no idtokenpayload.email', function(
-      done
-    ) {
-      this.auth0.getSSOData(function(err, result) {
-        expect(err).to.be(null);
-        expect(result).to.be.eql({
-          lastUsedConnection: { name: 'lastUsedConnection' },
-          lastUsedUserID: 'the-user-id',
-          lastUsedUsername: 'last-used-user-name',
-          lastUsedClientID: '...',
-          sessionClients: ['...'],
-          sso: true
+        this.webAuthSpy.checkSession.lastCall.args[1](null, {
+          idTokenPayload: { sub: 'the-user-id', name: 'last-used-user-name' }
         });
-        done();
       });
+      it('returns ssoData object with lastUsedConnection and idtokenpayload.email by default', function(
+        done
+      ) {
+        this.auth0.getSSOData(function(err, result) {
+          expect(err).to.be(null);
+          expect(result).to.be.eql({
+            lastUsedConnection: { name: 'lastUsedConnection' },
+            lastUsedUserID: 'the-user-id',
+            lastUsedUsername: 'last-used-user-email',
+            lastUsedClientID: '...',
+            sessionClients: ['...'],
+            sso: true
+          });
+          done();
+        });
 
-      this.webAuthSpy.checkSession.lastCall.args[1](null, {
-        idTokenPayload: { sub: 'the-user-id', name: 'last-used-user-name' }
+        this.webAuthSpy.checkSession.lastCall.args[1](null, {
+          idTokenPayload: {
+            sub: 'the-user-id',
+            email: 'last-used-user-email',
+            name: 'do not use me'
+          }
+        });
       });
     });
-    it('returns ssoData object with lastUsedConnection and idtokenpayload.email by default', function(
-      done
-    ) {
-      this.auth0.getSSOData(function(err, result) {
-        expect(err).to.be(null);
-        expect(result).to.be.eql({
-          lastUsedConnection: { name: 'lastUsedConnection' },
-          lastUsedUserID: 'the-user-id',
-          lastUsedUsername: 'last-used-user-email',
-          lastUsedClientID: '...',
-          sessionClients: ['...'],
-          sso: true
+    context('when inside of the hosted login page', function() {
```

Comment:
Since this is a different context, please separate it with 1 or 2 new lines.

Reply:
fixed